governance/members-and-charter.md

```diff
@@ -0,0 +1,109 @@
+# Governance Charter
+
+ Last update: March 2023
+
+ ## Overview
+
+ This document describes the process for the Keptn Governance Committee.
+ It describes the formation of the initial governance committee and its core responsibilities.
+ This includes its governance processes and processes to reform itself as necessary.
+
+
+ ## Governance Committee Scope
+
+
+ The Governance Committee has a set of rights and responsibilities including the following:
+
+ Charter for Technical Committee.
+ Define, evolve, and defend a Code of Conduct, which must include a neutral, unbiased process for resolving conflicts.
+ Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, etc.
+ Manage and organize the election process.
+ Logo/landing page/marketing.
+ Making a final decision if a consensus cannot be reached (see _Decision Making_)
+ Maintain relationships with the CNCF. For instance, creating documents describing the project
```

Comment:
Missing a full stop at the end. 

Generally it's quite a busy paragraph, might want to break it down a bit for readability